applications/HDF5Application/python_scripts/core/file_io.py

```diff
@@ -124,32 +124,32 @@ def Get(self, model_part):
 
 class _FilenameGetterWithDirectoryInitialization(object):
 
-    def __init__(self, settings):
+    def __init__(self, settings, data_comm):
```

Comment:
Is a deprecation message required here?

Reply:
I think not but lets hear @sunethwarna

Reply:
I dont think we have to have deprecation msgs since this is only a private method which is supposed to be used within this only. :)